ledger-core/virtual/object/object.go

```diff
@@ -56,6 +56,8 @@ type Info struct {
 	KnownRequests WorkingTable
 	// PendingTable holds requests that are known to be processed by other executors
 	PendingTable PendingTable
+	// ResultTable holds results that were executed
+	ResultTable ResultTable
```

Comment:
I don't think we need ResultTable. I think this should be discussed with the team

Reply:
@ruz Fixed